src/test/java/com/aws/greengrass/mqttclient/MqttClientTest.java

```diff
@@ -208,7 +208,7 @@ void afterEach() throws IOException {
     @ParameterizedTest
     @CsvSource({"10000,10000", "10000,10001"})
     void GIVEN_ping_timeout_gte_keep_alive_WHEN_mqtt_client_connects_THEN_throws_exception(int keepAlive,
-                                                                                          int pingTimeout) {
+                                                                                           int pingTimeout) {
```

Comment:
nit: is this spacing required?

Reply:
It was auto-formatted, I'll just put this on the previous line. I see that excessive length check is being suppressed here.